### DISCUSSION TITLE
Using Domoticz with "Home assistant option : on" in zigbee2mqtt

### DISCUSSION THREAD

**Discussion by mark-reijerkerk** [Koenkk/zigbee2mqtt#15923] [Show and tell]
In the "home assistant integration" tab. change the parameter "Home assistant advanced"  tab.  change the option for "home-discovery topic" in the too for example "zig-assistant"
After that make a new "MQTT Auto Discovery Client Gateway with LAN interface" in Domoticz with zig-assistant as auto discovery prefix.
After that you get all the information directly from Zigbee2mqtt, all the values are collected even the counters for example KWh and Voltage. 
The other way around  is going verry well too.
I don't know if someone is using this already but since I did not find a topic about I wanted to share it with the Community

All the devices are duped in Domoticz with the same friendly name as given in Zigbee2mqtt but with an other originator. After you disable or delete the plugin from Domoticz it you can see what happens. Greets MR

**Comment by mastadook** [comment]
thank you, this helped me!
any Idea how it works the other Way?
All Domoticz Devices to Homeassistant from zigbee2mqtt

**Comment by mark-reijerkerk** [comment]
Still investigating.

At moment I am monitoring a procedure do my first post via a second "Domoticz MQTT Auto Discovery Client"

One with the above mentioned **zig-assisant** prefix and one with the **zigbee2mqtt** prefix.

Everything is seamless at the moment but because there are duplicate Zigbee devices there are sync problems.
(when you select the wrong device after they are automatically imported by Domoticz)

You can use that too but be aware choose only the sub-devices from the **zig-assisant** hardware device.

Mark
